src/pages/Admin/constants.ts

```diff
@@ -40,3 +42,173 @@ export const proposalShape: SchemaShape<ProposalBase> = {
   title: stringByteSchema(4, 64),
   description: stringByteSchema(4, 1024),
 };
+
+type SIDEBAR_ROUTES = Exclude<
+  adminRoutes,
+  adminRoutes.proposal | adminRoutes.templates
+>;
+
+const _to: keyof Link = "to";
+
+export const LINKS: { [key in SIDEBAR_ROUTES]: Link & { [_to]: key } } = {
```

Comment:
nit: could convert `adminRoutes` to just `{ } as const` so could just access with "string" 
```ts
const link  = LINKS.withdraw // or ["withdraw"]
```

Reply:
Working on updating route logic to accommodate the `Liquid/Locked Account` pages.